src/controllers/connection/newConnection.ts

```diff
@@ -121,22 +122,53 @@ export class NewConnectionController extends HTMLController {
     }
 
     const inviteOrError = await this.decodeInvite(invite)
+
     if (inviteOrError.type === 'error') {
       return this.receiveInviteErrorHtml(inviteOrError.message)
     }
+
     const company = inviteOrError.company
 
     return this.html(
       this.fromInvite.fromInviteForm({
         feedback: {
           type: 'companyFound',
-          company: company,
+          company,
         },
         formStage: 'invite',
       })
     )
   }
 
+  /**
+   * @returns a feedback from a validated connection invitation
+   */
+  @SuccessResponse(200)
+  @Post('/verify-invite')
+  public async verifyInvite(
+    @Body()
+    body: {
+      invite: BASE_64_URL
+    }
+  ) {
+    // TODO update state so 'pending us/yours' and enable action button
+    const decodedInvite = await this.decodeInvite(body.invite)
+    // and other udates
+    if (decodedInvite.type === 'error') {
+      return this.newInviteErrorHtml('unable to decode invite request')
+    }
+
+    return this.html(
+      this.fromInvite.fromInviteForm({
+        feedback: {
+          type: 'success',
+          message: 'Please enter a 6 digit PIN code',
+        },
+        formStage: 'pin',
+      })
+    )
+  }
+
```

Comment:
As far as I can tell this is unused? What's this methoid supposed to do?

Reply:
going to investigate and will confirm shortly. i think this is form POST and get is only checking of changes (not 100% sure just yet) will get back on this one